spec/units/commands/internal/chdir_spec.rb

```diff
@@ -6,14 +6,14 @@
 
   describe '#execute' do
     it 'returns true for correct directories' do
-      env = stub(:[]= => true, puts_error: true)
+      env = double("Environment", :[]= => true, puts_error: true)
```

Comment:
This project prefers single quotes (it was started before we changed the style guide to recommend double quotes).